service/template.go

```diff
@@ -0,0 +1,33 @@
+package service
+
+import (
+	"context"
+
+	"release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type TemplateService struct {
+	repository model.TemplateRepository
+}
+
+func (s *TemplateService) Create(ctx context.Context, t model.Template) (model.Template, error) {
+	return s.repository.Insert(ctx, t)
+}
```

Comment:
The service should probably receive userID or orgID to only return templates that the user is authorized to access. Same applies to other endpoints. If I am not mistaken, the authorization is missing from all of them.